src/index.css

```
@@ -174,6 +218,38 @@
   line-height: 1.5;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  --radius: 0.625rem;
```

Comment for clear separation

```suggestion
  /* UDS Base Variables - Light Theme */
  --radius: 0.625rem;
```